File: day17.java
import java.util.Scanner;
class day17
{
    Scanner sc = new Scanner (System.in);
    static int t;
    int n , jar1, jar2, jar3;
    boolean flag = false;
    public static void main(String[] args)      // calls other methods of class
    {
        day17 ob = new day17();     // creating object to call static methods
        ob.test();
        for(int i=0; i <t; i++)     // to input and output t times
        {
            ob.input();
            ob.calculate();
            ob.output();
        }

    }
    void test()     // inputs value of t
    {
        System.out.print("Enter number of test cases: ");
        while(!flag)        // ensure input is within range
        {
            t = sc.nextInt();
            if(t>=1 && t<=100000)
                break;
            else
                System.out.print("Error!\nEnter number of test range within 1 to 10000");
        }
    }
    void input()        // inputs value of n
    {
        System.out.print("Enter number of chocolates: ");
                while(!flag)        // ensures input is within range
                {
                    n = sc.nextInt();
                    if(n>=4 && n<=100000)
                        break;
                    else
                        System.out.print("Error!\nEnter number of chocolates within 4 to 100000: ");
                }
    }
    void calculate()        // calculates the distribution of chocolates
    {
        if(n%2!=0)
        {
            jar1 = 1;       // jar1 has odd chocolates
            jar2 = (n-1)/2;
            jar3 = jar2;
            // jar2 and jar3 has same number of chocolates
        }
        else
        {
            jar1 = 1;
            jar2 = 1;
            // jar1 and jar2 has same number of chocolates
            jar3 = n-2;     // jar3 has odd number of chocolates
        }
    }
    void output()
    {
        System.out.println(jar1 + "," + jar2 + "," + jar3);
    }
}

File: day17.c
#include <stdio.h>
#include <stdbool.h>
void main()
{
    bool flag = false;
    int t, n;
    printf("Enter number of test case: ");
    while(flag==false)
    {
        scanf("%d", &t);
        if(t>=1 && t<=100)
            break;
        else
            printf("Error!\nEnter number of test cases wihtin 1 to 100: ");
    }
    for(int i=0; i<t; i++)
    {
        printf("Enter seat number: ");
        while(flag==false)
        {
            scanf("%d", &n);
            if(n>=1 && n<=30)
                break;
            else
                printf("Error\nEnter seat number from 1 to 30: ");
        }
        if(n>=1 && n<=10)
            printf("Lower Double\n");
        else if(n>=11 && n<=15)
            printf("Lower Single\n");
        else if(n>=16 && n<=25)
            printf("Upper Double\n");
        else if(n>=26 && n<=30)
            printf("Upper Single\n");
    }
}